### ISSUE TITLE
Paraview

### ISSUE THREAD

**Issue by gavalentine** [ubccr/software-layer#105]
Please install the latest version of Paraview for use via the updated OnDemand platform at CCR.

**Comment by dsajdak** [comment]
Paraview has been published to ccrsoft/2023.01

```
[vortex:~]$ module spider paraview

----------------------------------------------------------------------------------------------------------------
  paraview: paraview/5.11.2
----------------------------------------------------------------------------------------------------------------
    Description:
      ParaView is a scientific parallel visualizer.

    Properties:
      Visualisation software

    You will need to load all module(s) on any one of the lines below before the "paraview/5.11.2" module is available to load.

      gcc/11.2.0  openmpi/4.1.1

    Help:

      Description
      ===========
      ParaView is a scientific parallel visualizer.


      More information
      ================
       - Homepage: https://www.paraview.org


[vortex:~]$ module load gcc/11.2.0  openmpi/4.1.1 paraview/5.11.2
```

**Comment by gavalentine** [comment]
groovy!  thanks!

________________________________
